c-dependencies/js-compute-runtime/Makefile

```diff
@@ -23,9 +23,11 @@ SM_OBJ := $(SM_SRC)lib/*.o
 SM_OBJ += $(SM_SRC)lib/*.a
 FSM_SRC := $(ROOT_SRC)/js-compute-runtime/
 
+WASI_CC ?= /opt/wasi-sdk/bin/clang
 WASI_CXX ?= /opt/wasi-sdk/bin/clang++
+WIT_BINDGEN ?= ~/src/bytecodealliance/wit-bindgen/target/release/wit-bindgen
```

Comment:
Could we update the default path to be similar to where wasi-sdk is stored?
Hmm, another idea... if `WIT_BINDGEN` is not set, we could do this:
```
WIT_BINDGEN := $(shell which wit-bindgen)

ifndef WIT_BINDGEN
     $(error "No wit-bindgen in $PATH, consider doing cargo install --git https://github.com/bytecodealliance/wit-bindgen wit-bindgen-cli")
endif
```

That way we allow the user to store wit-bindgen anywhere they want to and we return a helpful error message which includes how to install wit-bindgen